src/Xamarin.Android.Build.Tasks/Tasks/Aot.cs

```diff
@@ -326,7 +326,7 @@ static int GetNdkApiLevel(string androidNdkPath, string androidApiLevel, Android
 						ldFlags,
 						QuoteFileName (SdkBinDirectory),
 						QuoteFileName (outdir),
-						sequencePointsMode == SequencePointsMode.Offline ? string.Format("gen-seq-points-file={0},", seqpointsFile) : string.Empty
+						sequencePointsMode == SequencePointsMode.Offline ? string.Format("msym-dir={0},", outdir) : string.Empty
```

Comment:
Should this be quoted? `outdir` sounds like a filesystem path, and thus should be quoted.